core-primitives/node-api-extensions/src/metadata.rs

```diff
@@ -0,0 +1,190 @@
+/*
+	Copyright 2021 Integritee AG and Supercomputing Systems AG
+
+	Licensed under the Apache License, Version 2.0 (the "License");
+	you may not use this file except in compliance with the License.
+	You may obtain a copy of the License at
+
+		http://www.apache.org/licenses/LICENSE-2.0
+
+	Unless required by applicable law or agreed to in writing, software
+	distributed under the License is distributed on an "AS IS" BASIS,
+	WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+	See the License for the specific language governing permissions and
+	limitations under the License.
+
+*/
+
+use crate::error::{Error, Result};
+use codec::{Decode, Encode};
+use sp_core::storage::StorageKey;
+use substrate_api_client::{Metadata, MetadataError};
+
+/// Handle SGX compatible substrate chain metadata.
+
+///Pallet's names
+pub static TEEREX: &str = "Teerex";
+pub static SIDECHAIN: &str = "Sidechain";
+pub static SYSTEM: &str = "Sidechain";
```

Comment:
Do these have to be `pub`? I would expect them to be only accessible in this module here.

Reply:
You're right and static SYSTEM: &str should be  "System";